mautic/mautic
# [UX] "Invite your team" action in account dropdown

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR takes other marketing tools in the market as reference to bring more relationship between different parts of Mautic.

![image](https://github.com/mautic/mautic/assets/116097999/6371a884-3d66-4828-9f50-330f1bbb8b23)

It adds a link to invite new users, that brings you to the create a new user screen if your role has permission to do so. Small improvement, but feels nice.

Hubspot, Mailchimp and others seek to provide a collaborative environment for companies. Everything becomes easier if you have more people joining the platform to configure and test different features.

![image](https://github.com/mautic/mautic/assets/116097999/e06fdaf9-a027-4a6a-b84d-3e5b0dbc8413)








---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Look at the top navbar and click to open the account dropdown
3. See the option and click
